### PR TITLE
docs: mocks

### PR DESCRIPTION
**Pull request by annipi**
Natspec documentation for contracts on mocks folder

### CODE REVIEW

File: contracts/mocks/ACME.sol [hunk 1]
```diff
@@ -4,6 +4,13 @@ pragma solidity ^0.8.16;
 import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
 
+/**
+ * @title ACME
+ * @dev ACME is a mock o f a service provider of lending
```
Comment (dvrosalesm) [inline contracts/mocks/ACME.sol:9]:
```suggestion
 * @dev ACME is a mocked service provider for lending
```

File: README.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
     <img src="https://www.rifos.org/assets/img/logo.svg" alt="logo" height="100" >
 </p>
 <h3 align="middle">RIF Everyday Defi Gateway</h3>
-<p align="middle">
-    RIF Everyday Defi Gateway
+<p>
+    The RIF Defi Gateway serves as a decentralized intermediary connecting Defi Protocol Service Providers, Consumers, and Wallets.  The key focus is delivering a service/protocol which can support target ecosystems in Everyday Defi.
```
Comment (dvrosalesm) [inline README.md:6]:
```suggestion
    The RIF DeFi Gateway serves as a decentralized intermediary protocol for connecting DeFi Protocol Service Providers, Consumers, and Wallets.  The key focus is delivering a service/protocol which can support target ecosystems in Everyday DeFi.
```

File: contracts/mocks/ACME.sol [hunk 3]
```diff
@@ -139,6 +235,13 @@ contract ACME is Ownable {
         _repay(currency, amount, payer, loaner);
     }
 
+    /**
+     * @notice Allows the payer to repay the debt of the loaner plus interest
+     * @param currency The currency of the loan
+     * @param amount The amount of currency to repay
+     * @param payer The address that will pay the debt of the loaner the currency
```
Comment (dvrosalesm) [inline contracts/mocks/ACME.sol:242]:
```suggestion
     * @param payer The address that will pay the debt of the loaner
```

**Review by dvrosalesm — APPROVED** [review]
LG!

**Review by davidfragalaureano — APPROVED** [review]
LGTM, nice work.

**Pull request merged**